db/Controllers/api-controller.js

```diff
@@ -0,0 +1,8 @@
+const api = require("../../endpoints.json")
+
+function getApi(req, res, next){
+    // console.log(api, "api in controller");
+    res.status(200).send(api)
```

Comment:
you should send back all your data from the controller on a meaningful key for example here something like {endpoints: api}